apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -285,13 +285,7 @@ def prepare_tag(
             )
 
         if format is FormatType.CODE:
-            inner_html = (
-                "<pl-code"
-                + (' language="' + code_language + '"' if code_language else "")
-                + ">"
-                + inner_html
-                + "</pl-code>"
-            )
+            inner_html = f'<pl-code {f'language="{code_language}"' if code_language else ""}>{inner_html}</pl-code>'
```

Comment:
You can't have a `'` inside an `f''` string. This causes an issue.

Reply:
ya, I was just fighting with that...I think i got it figured out.

Reply:
nevermind. appartently having nested f-strings, or f-strings when you have quotation marks inside them, is just a nightmare. I'm just going to give up on this change entirely, and open a new PR that just has the other fix.

Reply:
once we update to a python version that supports backslashes inside of f-strings, we should be able to make this work by having the inner f-string use different quotes than the outer f-string, and then having the quotation marks inside the inner f-string escaped with backslashes